flang/CMakeLists.txt

```diff
@@ -81,12 +81,13 @@ if (FLANG_STANDALONE_BUILD)
     mark_as_advanced(LLVM_ENABLE_ASSERTIONS)
   endif()
 
-  # We need a pre-built/installed version of LLVM.
-  find_package(LLVM REQUIRED HINTS "${LLVM_CMAKE_DIR}")
```

Comment:
This use of `LLVM_CMAKE_DIR` does not make much sense to me, because it is set by `LLVMConfig.cmake` which is not in effect before this `find_package` command executes.